content/en/cosign/verify.md

```diff
@@ -76,6 +76,14 @@ Verify image with local certificate and local certificate chain:
 $ cosign verify --cert cosign.crt --cert-chain chain.crt user/demo
 ```
 
+## Keyless verification using only a provided certificate chain with non-Fulcio roots
+Verify image with the provided certificate chain and identity parameters (intended for
+a "bring your own PKI" use case):
+
+```shell
+$ cosign verify --cert-chain chain.crt --certificate-oidc-issuer https://issuer.example.com --certificate-identity foo@example.com user/demo
```

Comment:
Can you update to certificate-chain here and fix the above example too? I like to avoid the abbreviation.